docs/help/Public_Resources.rst

```diff
@@ -51,5 +53,5 @@ whatever manner is easiest for you.
 .. _SIMP Users Mailing List: https://groups.google.com/forum/?fromgroups#!forum/simp-users
 .. _SIMP Developers List: https://groups.google.com/forum/?fromgroups#!forum/simp-dev
 .. _SIMP Announcement List: https://groups.google.com/forum/?fromgroups#!forum/simp-announce
-.. _SIMP Security List: https://groups.google.com/forum/?fromgroups#!forum/simp-security
+.. _SIMP Security List: mailto://simp-security@googlegroups.com
```

Comment:
I don't think this link is helpful. I think the email address should be mentioned inline with the reference to this link.